Updated documentation regarding catchup (#880)
Concordium/concordium.github.io: source/mainnet/net/nodes/node-requirements.rst

For node runners using Grafana®, Concordium provides a node performance dashboard using the exposed Prometheus metrics. You can `download it from the Grafana marketplace <https://grafana.com/grafana/dashboards/18983-concordium-node-external/>`__.

Synchronize a node with the network
===================================

When you start a node for the first time, it can take a while to synchronize
the node with the rest of the network, since it has to get all blocks from
its peers.

The startup time of the node can be improved by downloading the blocks from
an out-of-band catchup service before starting the node. While it will still
take time to process the blocks, it will typically be faster than requesting
them from peers.

> This feature is enabled by default since node version 6 and is controlled

Reviewer: ```suggestion
This feature is enabled by default in all operating systems since node version 6 and is controlled
```

> by the ``CONCORDIUM_NODE_CONSENSUS_DOWNLOAD_BLOCKS_FROM`` environment variable.

Reviewer: ```suggestion
by the ``CONCORDIUM_NODE_CONSENSUS_DOWNLOAD_BLOCKS_FROM`` environment variable in the configuration file.
```

For node runners using Grafana®, Concordium provides a node performance dashboard using the exposed Prometheus metrics. You can `download it from the Grafana marketplace <https://grafana.com/grafana/dashboards/18983-concordium-node-external/>`__.

Synchronize a node with the network
===================================

When you start a node for the first time, it can take a while to synchronize
the node with the rest of the network, since it has to get all blocks from
its peers.

The startup time of the node can be improved by downloading the blocks from
an out-of-band catchup service before starting the node. While it will still
take time to process the blocks, it will typically be faster than requesting
them from peers.

> This feature is enabled by default in all operating systems since version 6.1

Reviewer 2: ```suggestion
This feature is enabled by default in all distributions since version 6.1
```
